sdk/core/Azure.Core/tests/TestFramework/Azure.Core.TestFramework.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <DefineConstants>$(DefineConstants);TESTFRAMEWORK</DefineConstants>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Azure.Identity" />
+    <PackageReference Include="nunit" />
+    <PackageReference Include="NUnit3TestAdapter" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" />
+    <PackageReference Include="Moq" />
+  </ItemGroup>
+  <!-- Import Azure.Core shared source -->
+  <ItemGroup>
+    <Compile Include="$(AzureCoreSharedSources)EventSourceEventFormatting.cs" Link="Shared\EventSourceEventFormatting.cs" />
+    <Compile Include="$(AzureCoreSharedSources)ContentTypeUtilities.cs" Link="Shared\ContentTypeUtilities.cs" />
```

Comment:
Nit - If you rename the file in VS, does it update the `Link` accordingly?  Maybe do the `Link="Shared\%(RecursiveDir)\%(Filename)%(Extension)"` thing instead.